spec/view_models/admin/edit_auction_view_model_spec.rb

```diff
@@ -0,0 +1,93 @@
+require 'rails_helper'
+
+describe Admin::EditAuctionViewModel do
```

Comment:
these are tested at the integration level but we didn't have anything that hit these edge case so thought I'd throw in some unit tests!